### DISCUSSION TITLE
Damaged archive with posts while tranferring account

### DISCUSSION THREAD

**Discussion by I8ie** [mastodon/mastodon#23239] [General]
Hi, I was transferring my account to another server and while importing all my posts, it said that my archive is damaged and could not import. I tried to make another one with the same resault. Is there a way to repair the archive, make a new one that actually works or transfer my posts another way? I would appriecieate your help.

**Comment by ClearlyClaire** [comment]
Hi! There is currently no support in Mastodon for importing your old posts, so I am not sure what you are trying to do! You can import the list of people you follow, block or mute, but those are different from the archive that contains your posts.
